### ISSUE TITLE
why input_image_shape = K.placeholder(shape=(2, )) in retrain.py?

### ISSUE THREAD

**Issue by jianing-sun** [allanzelener/YAD2K#125]
Hi I'm trying to retrain a yolo model based on UNIMIB2016. I think the input image shape should be the model size which in this case should be (416., 416.). Why in the script it is (2,)? 
Another question, I run the code to do training, but the loss in validation set is always around 4000 and even if the loss goes down in the training set, it does have any improvement in the validation set. Does anyone know why is that?

Thanks!

**Comment by sun-su** [comment]
Hi, for the one, the input image shape is the dimension of (416, 416), so it is (2, ). Actually, it should be the original size of the image, so it may not be (416, 416), maybe (768, 516) for example.
